PresentScreenings/Controls/RatingField.cs

```diff
@@ -7,6 +7,7 @@ public class RatingField : NSTextField
     {
         #region Properties
         public FilmRatingDialogController DialogController { get; }
+        private NSCell CocoaCloseButton => (NSCell)DialogController.View.Window.AccessibilityCloseButton;
```

Comment:
Good thing to apply the discovered technique here too!
Please consider to use one method for both (and future) cases.

Reply:
Created #113 for this.